webapp/channels/src/components/admin_console/admin_definition.tsx

```diff
@@ -1130,6 +1130,191 @@ const AdminDefinition: AdminDefinitionType = {
                     ],
                 },
             },
+            export_storage: {
+                url: 'environment/export_storage',
+                title: defineMessage({id: 'admin.sidebar.exportStorage', defaultMessage: 'Export Storage'}),
+                isHidden: it.any(
+                    it.not(it.licensedForFeature('Cloud')),
+                    it.not(it.licensedForSku(LicenseSkus.Enterprise)),
+                    it.configIsFalse('FeatureFlags', 'CloudDedicatedExportUI'),
+                ),
+                schema: {
+                    id: 'ExportFileSettings',
+                    name: defineMessage({id: 'admin.sidebar.exportStorage', defaultMessage: 'Export Storage'}),
+                    settings: [
+                        {
+                            type: 'bool',
+                            key: 'FileSettings.DedicatedExportStore',
+                            label: defineMessage({id: 'admin.exportStorage.dedicatedExportStore', defaultMessage: 'Enable Dedicated Export Store:'}),
+                            help_text: defineMessage({id: 'admin.exportStorage.dedicatedExportStoreDescription', defaultMessage: 'When enabled, Mattermost will use a dedicated export storage bucket for all export operations. This is required for Mattermost Cloud deployments.'}),
+                            isDisabled: it.not(it.userHasWritePermissionOnResource(RESOURCE_KEYS.ENVIRONMENT.FILE_STORAGE)),
+                        },
+                        {
+                            type: 'dropdown',
+                            key: 'FileSettings.ExportDriverName',
+                            label: defineMessage({id: 'admin.exportStorage.exportDriverName', defaultMessage: 'Export Storage Driver:'}),
+                            isDisabled: true,
+                            isHidden: it.stateEquals('FileSettings.DedicatedExportStore', false),
+                            options: [
+                                {
+                                    value: FILE_STORAGE_DRIVER_S3,
+                                    display_name: defineMessage({id: 'admin.image.storeAmazonS3', defaultMessage: 'Amazon S3'}),
+                                },
+                            ],
+                        },
```

Comment:
From the screenshot (https://private-user-images.githubusercontent.com/12176405/299415429-eeca272b-0c1e-41c9-bf6e-2cbb77ff5652.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYyNjc2NDAsIm5iZiI6MTcwNjI2NzM0MCwicGF0aCI6Ii8xMjE3NjQwNS8yOTk0MTU0MjktZWVjYTI3MmItMGMxZS00MWM5LWJmNmUtMmNiYjc3ZmY1NjUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTI2VDExMDkwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY5Nzc2ZjUyYjk0ODA2NWZiY2MwMzAxMzhlZTJhNzQyNmE5ZmJhZGU0MzUyMDExZjgxMTg4OTk0NTAwNmVlMzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.GXKayGa_BAWcf8vyCJIy_UIxRf5r6WKlRrlvPj5eNuw) is doesn't seem like the dropdown is disabled. I was expecting a dark background.

Is the dropdown really disabled?

Reply:
![image](https://github.com/mattermost/mattermost/assets/12176405/45e71529-3794-4ee9-9824-97e5a86e4313)
One of the pictures was before we decided to not have a NONE option. this is what that dropdown will look like